translations/messages.en.xlf

```diff
@@ -169,6 +177,14 @@
                 <source>action.browse_admin</source>
                 <target>Browse backend</target>
             </trans-unit>
+            <trans-unit id="action.edit_user">
+                <source>action.edit_user</source>
+                <target>Edit user</target>
+            </trans-unit>
+            <trans-unit id="action.change_password">
+                <source>action.change_password</source>
+                <target>Change password</target>
+            </trans-unit>
```

Comment:
The `action.update_user_info` is missing.

Reply:
Actually, it was the translation key in the template file that was wrong.